packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -21,21 +20,14 @@ function InserterListItem( {
 		backgroundColor: icon.background,
 		color: icon.foreground,
 	} : {};
-	const itemIconStackStyle = icon && icon.shadowColor ? {
```

Comment:
Does this remove the support for coloring icons entirely? Or just coloring the stack?

Reply:
just the stack